@lingosta/app/constants.ts

```diff
@@ -2,6 +2,8 @@ export const SUPPORT_EMAIL: string = "hello@lingosta.app";
 
 const APP_NAME: string = "Lingosta";
 
+const LINGOSTA_GREEN: string = 'teal-500';
```

Comment:
you'd actually want to implement this in the code as well 😄 but that's ok for now. Let's just merge this in since the deadline is coming.